similarity/operations.py

```diff
@@ -0,0 +1,102 @@
+from sqlalchemy import text
+
+
+class Metric(object):
+    name = ''
+    description = ''
+    category = ''
+
+    def __init__(self, connection):
+        self.connection = connection
+
+    def _create(self, hybrid, column):
+        hybrid = str(hybrid).upper()
+        metrics_query = text("""
+            INSERT INTO similarity_metrics (metric, is_hybrid, description, category, visible)
+                 VALUES (:metric, :hybrid, :description, :category, TRUE)
+            ON CONFLICT (metric)
+          DO UPDATE SET visible=TRUE
+        """)
+        self.connection.execute(metrics_query, {'metric': self.name,
+                                                'hybrid': hybrid,
+                                                'description': self.description,
+                                                'category': self.category})
+
+        # This can be removed if not using postgres similarity solution
```

Comment:
we don’t need to keep commented code - we have a git history available which we can restore from if needed. you can delete this